docker-compose.yml

```diff
@@ -64,10 +67,24 @@ services:
     image: memcached:latest
 
   tinyproxy:
-    image: mozillabteam/tinyproxy
+    build:
+      context: docker/tinyproxy
+      dockerfile: ./Dockerfile
     ports:
-      - "1091:1090"
+      - "1080:1080"
 
 volumes:
   bmo-mysql-db:
   bmo-data-dir:
+  bmo-sync-dir:
```

Comment:
You might not need to separately define these volumes out here: I had to do it in `suite` because `volumes_from` was deprecated, and it was the only way to share a host-mount volume between two containers.
However, if the mount is only needed for one container, you should still be able to use the shorthand in the service definition:
```
  bmo-web.vm:
    build: &bmo_build
      context: .
      dockerfile: Dockerfile
    command: dev_httpd
    volumes:
      - bmo-data-dir:/app/data
      - .:/mnt/sync
```

Reply:
Thanks. Now I realize when you need to do that.